index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link rel="stylesheet" href="style.css" type="text/css"> -->
+    <link rel="stylesheet" href="style1.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+<section>
+    <header>
+        <nav class="nav-bar">
+            <a href="#" class="nav-title">Welcome</a>
+            <a href="#" class="hamburger"><img src="Images/Enabled.svg" alt="hamburger"></a>
+        </nav>
+    </header>
+    <div class="hero">
+        <div class="hero-container">
+                <h1 class="hero-title"> Hey there. I’m Sana <br>
+                    <span class="hero-span">I’m a software developer</span>
+                </h1>
+            </div>
+        <div class="hero-container2">
+            <p class="hero-description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, <br> don’t hestiate to contact me.</p>
+        </div>
+
+        <div class="social">
+            <a class="icon" href="#"><img src="Images/twitter (3).svg" alt="twitter-img"></a>
+            <a class="icon" href="#"><img src="Images/Linkedin icon (1).svg" alt="linkedIn-img"></a>
+            <a class="icon" href="#"><img src="Images/medium (1).svg" alt="medium"></a>
+            <a class="icon" href="#"><img src="Images/github.svg" alt="github-img"></a>
+            <a class="icon" href="#"><img src="Images/angellist.svg" alt="angellist-img"></a>
+        </div>
+        <div class="action-icon">
+            <a href="#"><img class="action" src="Images/Disabled.svg" alt=""></a>
+        </div>
```

Comment:
- [ ] kindly consider the list of icons to use  a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.